openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -37,14 +41,31 @@ function download {
   done
 }
 
+function download_ingress {
+  component=$1
+  version=$2
+  shift
+  shift
+
+  files=("$@")
+
+  ingress_dir="$root/openshift-knative-operator/cmd/operator/kodata/ingress/$(versions.major_minor "${KNATIVE_SERVING_VERSION}")"
+  rm -r "$ingress_dir"
+  mkdir -p "$ingress_dir"
+
+  for (( i=0; i<${#files[@]}; i++ ));
+  do
+    index=$(( i+1 ))
+    file="${files[$i]}.yaml"
+    ingress_target_file="$ingress_dir/$index-$file"
+    url="https://raw.githubusercontent.com/openshift-knative/${component}/${version}/config/${file}"
```

Comment:
should we download from upstream?

the files on the TAGs there do have the proper `release` labels, e.g. see here: https://github.com/knative-sandbox/net-istio/releases/download/v0.21.0/net-istio.yaml

Reply:
See https://github.com/openshift-knative/serverless-operator/pull/924#discussion_r623650459. We **can**, but then have to deal with filtering this YAML to only the set we actually want.